dev-docs/RFCs/editable-layers/generic-edit-mode.md

```diff
@@ -115,21 +115,21 @@ class EditableGeoJsonLayer {
 
 ### Module layout
 
-We will need a `@nebula.gl/edit-modes` module separate from the `nebula.gl` module. The reason is because this new `@nebula/edit-modes` should have no deck.gl dependency.
+We will need a `@@deck.gl-community/editable-layers/edit-modes` module separate from the `@deck.gl-community/editable-layers` module. The reason is because this new `@nebula/edit-modes` should have no deck.gl dependency.
 
-- `nebula.gl`
-  - depends on `@nebula.gl/edit-modes`, `@nebula.gl/layers`, and all the other `@nebula/...` modules.
+- `@deck.gl-community/editable-layers`
+  - depends on `@@deck.gl-community/editable-layers/edit-modes`, `@@deck.gl-community/editable-layers/layers`, and all the other `@nebula/...` modules.
   - doesn't have much in it, just basically imports from the others and re-exports them
-- `@nebula.gl/edit-modes`
+- `@@deck.gl-community/editable-layers/edit-modes`
   - depends [turf.js](http://turfjs.org/), no (large) dependencies like deck.gl
   - contains all the modes for editing GeoJSON (e.g. `DrawPolygonMode`)
   - contains `EditMode` interface
   - contains other general purpose types and classes (e.g. event types like `ClickEvent`)
   - this module will be reused by `react-map-gl-draw`
-- `@nebula.gl/layers`
-  - depends on `@nebula.gl/edit-modes` and `deck.gl`
+- `@@deck.gl-community/editable-layers/layers`
+  - depends on `@@deck.gl-community/editable-layers/edit-modes` and `deck.gl`
   - contains `EditableGeoJsonLayer`, a deck.gl `CompositeLayer`
-- Other modules are unaffected (e.g. `@nebula.gl/overlays`)
+- Other modules are unaffected (e.g. `@@deck.gl-community/editable-layers/overlays`)
```

Comment:
See above. :)

Reply:
Yes, I think we should restore these RFCs from the original nebula repo, or remove them.